plugins/Installation/lang/en.json

```diff
@@ -19,145 +19,145 @@
         "SystemCheckFileIntegrity": "File integrity",
-        "SystemCheckFilterHelp": "You need to configure and rebuild PHP with \"filter\" support enabled (don't use --disable-filter).",
+        "SystemCheckFilterHelp": "You need to configure and rebuild PHP with \"filter\" support (don't use --disable-filter).",
         "SystemCheckFunctions": "Required functions",
-        "SystemCheckShellExecHelp": "We recommend you enable this PHP built-in function. %1$sRead this to learn more.%2$s",
-        "SystemCheckGDFreeType": "GD > 2.x + Freetype (graphics)",
-        "SystemCheckGDHelp": "The sparklines (small graphs) and image graphs (in Matomo Mobile app and Email reports) will not work.",
-        "SystemCheckGlobHelp": "This built-in function has been disabled on your host. Matomo will attempt to emulate this function but may encounter further security restrictions. Functionality may be impacted.",
-        "SystemCheckGzcompressHelp": "You need to enable the zlib extension and gzcompress function.",
+        "SystemCheckShellExecHelp": "It is recommended to turn on this PHP built-in function. %1$sRead this to learn more.%2$s",
+        "SystemCheckGDFreeType": "GD > 2.x + FreeType (graphics)",
+        "SystemCheckGDHelp": "The sparklines (small graphs) and image graphs (in the Matomo mobile app and email reports) will not work.",
+        "SystemCheckGlobHelp": "This built-in function has been turned off on your host. Matomo will attempt to emulate it, but may encounter further security restrictions. Functionality may be impacted.",
+        "SystemCheckGzcompressHelp": "You need to turn on the zlib extension and gzcompress function.",
         "SystemCheckGzuncompressHelp": "You need to enable the zlib extension and gzuncompress function.",
-        "SystemCheckHashHelp": "You need to configure and rebuild PHP with hash() support enabled by excluding the option --disable-hash.",
-        "SystemCheckJsonHelp": "The php-json extension is required for Matomo to read and write JSON data.",
-        "SystemCheckMailHelp": "Feedback and Lost Password messages will not be sent without mail().",
+        "SystemCheckHashHelp": "Configure and rebuild PHP with required hash() support by excluding the option --disable-hash.",
+        "SystemCheckJsonHelp": "The required php-json extension is needed by Matomo to read and write JSON data.",
+        "SystemCheckMailHelp": "Feedback and 'Lost password' messages will not be sent without 'email()'.",
         "SystemCheckMemoryLimit": "Memory limit",
-        "SystemCheckMemoryLimitHelp": "On a high traffic website, the archiving process may require more memory than currently allowed. If necessary, change the memory_limit directive in your php.ini file.",
+        "SystemCheckMemoryLimitHelp": "On a high-traffic website, the archival process may require more memory than currently allowed. If necessary, change the memory_limit directive in your php.ini file.",
```

Comment:
Suggest reverting this one, not adding clarity or fixing any language issues.

Reply:
It isn't the process archiving (wrong tense), it is the process for archival(?).